chain/client/src/client.rs

```diff
@@ -2516,8 +2512,6 @@ impl Client {
             .iter()
             .filter_map(|ShardInfo(shard_id, _)| self.should_split_shard(shard_id, me, prev_hash))
             .collect();
-        // For colour decorators to work, they need to printed directly. Otherwise the decorators get escaped, garble output and don't add colours.
-        debug!(target: "catchup", progress_per_shard = ?format_shard_sync_phase_per_shard(&shards_to_split, false), "Need to split states for shards");
```

Comment:
This log line doesn't make much sense. 
- It is called multiple times, including after resharding is started but it doesn't actually know about the current status of it. 
- There is a different log line just a bit later that prints the current status anyway.